cmd/generate/config/rules/harness.go

```diff
@@ -0,0 +1,44 @@
+package rules
+
+import (
+	"regexp"
+
+	"github.com/zricethezav/gitleaks/v8/cmd/generate/secrets"
+	"github.com/zricethezav/gitleaks/v8/config"
+)
+
+func HarnessPAT() *config.Rule {
```

Comment:
bringing this up for completeness sake - what do you think of having a single `Harness-Api-Key` for `(pat|sat)\...` instead of splitting it into two?

Reply:
Updated via https://github.com/gitleaks/gitleaks/pull/1406/commits/a1d989dc32ac7164395d825b8c0f14d324e0daa2